Config/Inc/config.h

```diff
@@ -101,8 +102,12 @@ typedef enum SafetyStatus_e {SAFE = 0, DANGER = 1, OVERVOLTAGE = 2, UNDERVOLTAGE
 #define MAX_VOLT_SENSORS_PER_MINION_BOARD	12	// User defined. The LTC6811 can actually measure 12 modules.
 #endif
 
+static const uint8_t VOLT_TAP_DIST[NUM_MINIONS] = {11, 10, 11}; // the distribution of modules each minion is monitoring
```

Comment:
Im not sure about this being in the config.h file. If it works it works but I'm pretty sure declarations of a static array should be done in voltage.c. not config.h as a lot of files pull from here. We can also just define # of taps per module in config.h

Reply:
better way of doing this (instead of static) is
```
__attribute__((weak)) const int 
VOLT_TAP_DIST[] = {11, 10, 11}; // the distribution of modules each minion is monitoring
_Static_assert((sizeof(VOLT_TAP_DIST)/sizeof(*VOLT_TAP_DIST)) == NUM_MINIONS, "error: ur cfg wrong");
```

Reply:
agreed, this will check it before runtime right?